Documentation/configuration.md

```diff
@@ -1,44 +1,46 @@
 # rkt configuration
 
-`rkt` reads configuration from two directories - system one and local
-one. System directory is by default `/usr/lib/rkt` and local directory
-- `/etc/rkt`. `rkt` looks for configuration files inside
-subdirectories of those two. Inside them, it ignores everything but
-regular files with `.json` extension. This means - `rkt` does _not_
-search for the files by recursively going down the directory
-tree. This also means that users are free to put some additional files
-there as a documentation.
+`rkt` reads configuration from two directories - a **system directory** and a
+**local directory**.
+The system directory defaults to `/usr/lib/rkt` and the local directory
+`/etc/rkt`, but both can be overridden at the command-line.
+
+`rkt` looks for configuration files inside subdirectories of these two
+directories. It ignores everything but regular files with `.json` extension.
+This means `rkt` does _not_ search for the files by recursively going down
+the directory tree. This also means that users are free to put some additional
+files there (e.g. documentation).
 
 Every configuration file has two common fields: `rktKind` and
 `rktVersion`. Both are strings. Rest of the fields are specified by
 that pair. Currently supported kinds and versions are described
 below. These fields have to be specified and cannot be empty.
 
-Kind describes the type of the configuration. This is to avoid putting
+`rktKind` describes the type of the configuration. This is to avoid putting
 unrelated values into single monolitic file.
 
-Version allows configuration kind versioning. Please note that the new
+`rktVersion` allows configuration kind versioning. Please note that the new
```

Comment:
I assume this is the `rkt` version, but it's a little unclear.
Could be interpreted as the config file schema version for this kind.